docs/GettingStarted/DockerComposeSetup.md

```diff
@@ -33,8 +33,87 @@ sidebar_position: 1
 
 Support for database schema migration was introduced to DevLake in v0.10.0. From v0.10.0 onwards, users can upgrade their instance smoothly to a newer version. However, versions prior to v0.10.0 do not support upgrading to a newer version with a different database schema.
 
-<br/>
+<br/> 
 
+## FAQ
+### 1. Can I use an external Database service instead of running Database in Docker?
+   
+    Yes, you can simply comment out mysql/grafana part in docker-compose.yml and update some configurations in .env befor you run `docker compose up -d`, here are the steps:
+
+1. Comment out mysql part: 
+```yaml
+  mysql:
+    image: mysql:8
+    volumes:
+      - mysql-storage:/var/lib/mysql
+    restart: always
+    ports:
+      - "127.0.0.1:3306:3306"
+    environment:
+      MYSQL_ROOT_PASSWORD: admin
+      MYSQL_DATABASE: lake
+      MYSQL_USER: merico
+      MYSQL_PASSWORD: merico
+    command:
+      --character-set-server=utf8mb4
+      --collation-server=utf8mb4_bin
+```
+
+2. Comment out mysql volume:
```

Comment:
Text change:

Comment out the 'mysql' volume:

Reply:
fixed